src/screens/Assets/CollectiblesList.js

```diff
@@ -127,6 +127,7 @@ class CollectiblesList extends React.Component<Props> {
     const { navigation } = this.props;
     return (
       <BadgeTouchableItem
+        background={`#${item.backgroundColor}`}
```

Comment:
Correct me if I'm wrong, but `backgroundColor` here is now always present as you also check for the same in in `AssetCardMinimized.js`?